File: CodeGenDotNet 159059-1372 (a)/CSharp/Chapter 1/HelloWorldAndSimpleClass/Test/ClassOrdersViaXSLT.cs

using System;

/// <summary>
/// 
/// </summary>

public class Orders
{
	#region Class level declarations
	private Int32 m_OrderID;

	private String m_CustomerID;

	private Int32 m_EmployeeID;

	private DateTime m_OrderDate;

	private DateTime m_RequiredDate;

	private DateTime m_ShippedDate;

	private Int32 m_ShipVia;

	private Decimal m_Freight;

	private String m_ShipName;

	private String m_ShipAddress;

	private String m_ShipCity;

	private String m_ShipRegion;

	private String m_ShipPostalCode;

	private String m_ShipCountry;

	#endregion

	#region Public Methods and Properties

	public Int32 OrderID
	{
		get
		{
			Return m_OrderID;
		}
		set
		{
			m_OrderID = value;
		}
	}

	public String CustomerID
	{
		get
		{
			Return m_CustomerID;
		}
		set
		{
			m_CustomerID = value;
		}
	}

	public Int32 EmployeeID
	{
		get
		{
			Return m_EmployeeID;
		}
		set
		{
			m_EmployeeID = value;
		}
	}

	public DateTime OrderDate
	{
		get
		{
			Return m_OrderDate;
		}
		set
		{
			m_OrderDate = value;
		}
	}

	public DateTime RequiredDate
	{
		get
		{
			Return m_RequiredDate;
		}
		set
		{
			m_RequiredDate = value;
		}
	}

	public DateTime ShippedDate
	{
		get
		{
			Return m_ShippedDate;
		}
		set
		{
			m_ShippedDate = value;
		}
	}

	public Int32 ShipVia
	{
		get
		{
			Return m_ShipVia;
		}
		set
		{
			m_ShipVia = value;
		}
	}

	public Decimal Freight
	{
		get
		{
			Return m_Freight;
		}
		set
		{
			m_Freight = value;
		}
	}

	public String ShipName
	{
		get
		{
			Return m_ShipName;
		}
		set
		{
			m_ShipName = value;
		}
	}

	public String ShipAddress
	{
		get
		{
			Return m_ShipAddress;
		}
		set
		{
			m_ShipAddress = value;
		}
	}

	public String ShipCity
	{
		get
		{
			Return m_ShipCity;
		}
		set
		{
			m_ShipCity = value;
		}
	}

	public String ShipRegion
	{
		get
		{
			Return m_ShipRegion;
		}
		set
		{
			m_ShipRegion = value;
		}
	}

	public String ShipPostalCode
	{
		get
		{
			Return m_ShipPostalCode;
		}
		set
		{
			m_ShipPostalCode = value;
		}
	}

	public String ShipCountry
	{
		get
		{
			Return m_ShipCountry;
		}
		set
		{
			m_ShipCountry = value;
		}
	}


	#endregion

}

File: CodeGenDotNet 159059-1372 (a)/VBCode/Chapter 1/HelloWorldAndSimpleClass/Test/ClassOrdersViaXSLT.vb

Option Strict On
Option Explicit On

Imports System

'! Class Summary: 

Public Class Orders
   
#Region "Class level declarations"
   Private m_OrderID As Int32
   Private m_CustomerID As String
   Private m_EmployeeID As Int32
   Private m_OrderDate As DateTime
   Private m_RequiredDate As DateTime
   Private m_ShippedDate As DateTime
   Private m_ShipVia As Int32
   Private m_Freight As Decimal
   Private m_ShipName As String
   Private m_ShipAddress As String
   Private m_ShipCity As String
   Private m_ShipRegion As String
   Private m_ShipPostalCode As String
   Private m_ShipCountry As String
#End Region

#Region "Public Methods and Properties"

   Public Property OrderID() As Int32
      Get
         Return m_OrderID
      End Get
      Set(ByVal Value As Int32)
         m_OrderID = Value
      End Set
   End Property

   Public Property CustomerID() As String
      Get
         Return m_CustomerID
      End Get
      Set(ByVal Value As String)
         m_CustomerID = Value
      End Set
   End Property

   Public Property EmployeeID() As Int32
      Get
         Return m_EmployeeID
      End Get
      Set(ByVal Value As Int32)
         m_EmployeeID = Value
      End Set
   End Property

   Public Property OrderDate() As DateTime
      Get
         Return m_OrderDate
      End Get
      Set(ByVal Value As DateTime)
         m_OrderDate = Value
      End Set
   End Property

   Public Property RequiredDate() As DateTime
      Get
         Return m_RequiredDate
      End Get
      Set(ByVal Value As DateTime)
         m_RequiredDate = Value
      End Set
   End Property

   Public Property ShippedDate() As DateTime
      Get
         Return m_ShippedDate
      End Get
      Set(ByVal Value As DateTime)
         m_ShippedDate = Value
      End Set
   End Property

   Public Property ShipVia() As Int32
      Get
         Return m_ShipVia
      End Get
      Set(ByVal Value As Int32)
         m_ShipVia = Value
      End Set
   End Property

   Public Property Freight() As Decimal
      Get
         Return m_Freight
      End Get
      Set(ByVal Value As Decimal)
         m_Freight = Value
      End Set
   End Property

   Public Property ShipName() As String
      Get
         Return m_ShipName
      End Get
      Set(ByVal Value As String)
         m_ShipName = Value
      End Set
   End Property

   Public Property ShipAddress() As String
      Get
         Return m_ShipAddress
      End Get
      Set(ByVal Value As String)
         m_ShipAddress = Value
      End Set
   End Property

   Public Property ShipCity() As String
      Get
         Return m_ShipCity
      End Get
      Set(ByVal Value As String)
         m_ShipCity = Value
      End Set
   End Property

   Public Property ShipRegion() As String
      Get
         Return m_ShipRegion
      End Get
      Set(ByVal Value As String)
         m_ShipRegion = Value
      End Set
   End Property

   Public Property ShipPostalCode() As String
      Get
         Return m_ShipPostalCode
      End Get
      Set(ByVal Value As String)
         m_ShipPostalCode = Value
      End Set
   End Property

   Public Property ShipCountry() As String
      Get
         Return m_ShipCountry
      End Get
      Set(ByVal Value As String)
         m_ShipCountry = Value
      End Set
   End Property


#End Region

End Class
